src/main/scala/com/microsoft/hyperspace/actions/RefreshIncrementalAction.scala

```diff
@@ -49,6 +51,13 @@ class RefreshIncrementalAction(
     logManager: IndexLogManager,
     dataManager: IndexDataManager)
     extends RefreshActionBase(spark, logManager, dataManager) {
+
+  override val fileIdTracker = {
+    val tracker = new FileIdTracker(previousIndexLogEntry.maxFileId)
```

Comment:
question. so the initial value will be the previous max file id?

Reply:
Yes, I renamed `startId` -> `lastId` in FileIdTracker for more clarification.

Reply:
`startId` seems better to me. My question was (sorry if I wasn't clear) is why can't this start from -1?

Reply:
No there is no requirement behind it. We should start from some number. We can start from -1 or even `Long.MinValue`. In the [Proposal](https://github.com/microsoft/hyperspace/issues/220), I proposed to start from 0 as it is clear, and the range of non-negative numbers is large enough to cover our source file id generation needs. So do you want to change the starting point? If yes, please let me know what number you want and I will set as Default_Max.